crates/re_renderer/src/renderer/mod.rs

```diff
@@ -52,45 +52,36 @@ pub trait Renderer {
     ) -> Self;
 
     // TODO(andreas): Some Renderers need to create their own passes, need something like this for that.
-    // TODO(andreas): The harder part is that some of those might need to share them with others!
-    //                E.g. Shadow Mapping! Conceivable that there are special traits for those (distinguish "ShadowMappingAwareRenderer")
-    // fn record_custom_passes<'a>(
-    //     &self,
-    //     pools: &'a WgpuResourcePools,
-    //     pass: &mut wgpu::CommandEncoder,
-    //     draw_data: &'a Self::DrawData,
-    // ) -> anyhow::Result<()> {
-    // }
 
+    /// Called once per phase given by [`Renderer::participated_phases`].
     fn draw<'a>(
         &self,
         pools: &'a WgpuResourcePools,
+        phase: DrawPhase,
         pass: &mut wgpu::RenderPass<'a>,
         draw_data: &'a Self::RendererDrawData,
     ) -> anyhow::Result<()>;
 
-    /// Relative location in the rendering process when this renderer should be executed.
-    /// TODO(andreas): We might want to take [`DrawData`] into account for this.
-    ///                But this touches on the [`Renderer::draw`] method might be split in the future, which haven't designed yet.
-    fn draw_order() -> u32 {
-        DrawOrder::Opaque as u32
+    /// Combination of flags indicating in which phases [`Renderer::draw`] should be called.
+    fn participated_phases() -> &'static [DrawPhase] {
+        &[DrawPhase::Opaque]
     }
 }
 
-/// Assigns rough meaning to draw sorting indices
-#[allow(dead_code)]
-#[repr(u32)]
-enum DrawOrder {
+/// Determines a (very rough) order of rendering and describes the active [`wgpu::RenderPass`].
+///
+/// Currently we do not support sorting *within* a rendering phase!
+/// See [#702](https://github.com/rerun-io/rerun/issues/702)
+#[derive(Clone, Copy, Hash, PartialEq, Eq)]
+pub enum DrawPhase {
     /// Opaque objects, performing reads/writes to the depth buffer.
+    ///
     /// Typically they are order independent, so everything uses this same index.
-    Opaque = 30000,
-
-    /// Transparent objects. Each draw typically gets its own sorting index.
-    Transparent = 50000,
+    Opaque,
 
-    /// Backgrounds should always be rendered last.
-    Background = 70000,
+    /// Background, rendering where depth wasn't written.
+    Background,
 
-    /// Postprocessing effects that are applied before the final tonemapping step.
-    Postprocess = 90000,
+    /// Drawn when compositing with the main target.
+    Composition,
```

Comment:
```suggestion
    Compositing,
```

composing != compositing :)